test/Elastic.Apm.Elasticsearch.Tests/ElasticsearchFixture.cs

```diff
@@ -32,9 +32,9 @@ public async Task InitializeAsync()
 
 		public async Task DisposeAsync()
 		{
-			await _container.StopAsync();
 			try
 			{
+				await _container.StopAsync();

```

Comment:
This throws an exception with:

```
[Test Class Cleanup Failure (Elastic.Apm.Elasticsearch.Tests.ElasticsearchTests)]: System.InvalidOperationException : Testcontainer has not been created.
Stack Trace:
   at DotNet.Testcontainers.Containers.TestcontainersContainer.ThrowIfContainerHasNotBeenCreated()
   at DotNet.Testcontainers.Containers.TestcontainersContainer.get_Id()
   at DotNet.Testcontainers.Containers.TestcontainersContainer.StopAsync(CancellationToken ct)
   at Elastic.Apm.Elasticsearch.Tests.ElasticsearchFixture.DisposeAsync() in C:\Users\jenkins\workspace\net_apm-agent-dotnet-mbp_PR-1826\apm-agent-dotnet\test\Elastic.Apm.Elasticsearch.Tests\ElasticsearchFixture.cs:line 35
```

But this happens after the test runs (it's in `DisposeAsync`) and I don't see an exception for the asserts in the real test that run against the container. This makes me think that maybe there is some issue during `Dispose`, but the test itself was successful.

So first attempt: let's ignore these exceptions in `DisposeAsync` and see if the test itself is green.